deltacat/storage/model/manifest.py

```diff
@@ -192,6 +194,20 @@ def of(
         manifest = Manifest._build_manifest(meta, entries, author, uuid)
         return manifest
 
+    @staticmethod
+    def from_json(json_string: str) -> Manifest:
+        parsed_dict = json.loads(json_string)
+        return Manifest.of(
+            entries=ManifestEntryList.of(
+                [
+                    ManifestEntry.from_dict(entry)
+                    for entry in parsed_dict.get("entries", [])
+                ]
+            ),
+            author=ManifestAuthor.from_dict(parsed_dict.get("author", {})),
```

Comment:
This behavior isn't fully consistent with the default constructor behavior of omitting the `author` key completely if it doesn't exist (this will instead set the `author` key to an empty dictionary).



```suggestion
            author=ManifestAuthor.from_dict(parsed_dict["author"]) if "author" in parsed_dict else None,
```

Reply:
addressed